src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -178,29 +183,40 @@ public function publish(string $body, array $headers = [], int $delay = 0): void
             $this->setup();
         }
 
-        $flags = $this->queueConfiguration['flags'] ?? AMQP_NOPARAM;
-        $attributes = $this->getAttributes($headers);
+        // TODO - allow flag & attributes to be configured on the message
```

Comment:
@sroze Do you have any thoughts about this part? I made this change, because I didn't think it made sense for the `flags` from `queueConfiguration` to be used when sending a message (queues and publishing are 2 totally separate concepts, right?). Same thing with `attributes`, which also came from `queueConfiguration`. Does this make sense to you? And do these things need to be configurable - e.g. by adding another stamp or making the `AmqpRoutingKeyStamp` a more generic `AqmpTransportStamp`?

Reply:
I don't believe that they need to be configurable per message to be honest, I'm not sure if there is such use-case. Therefore, I'd get rid of these TODOs 😉